### ISSUE TITLE
Gem version 0.2.4 missing from rubygems

### ISSUE THREAD

**Issue by eliotsykes** [wistia/wistia-api#29]
The most recent version available on rubygems is 0.2.3 from 2013. 0.2.4 is missing:

https://rubygems.org/gems/wistia-api/versions

**Comment by kylefox** [comment]
@lpommers Is this gem abandoned?

**Comment by lpommers** [comment]
@kylefox yeah we are not currently maintaining/updating this gem. 

Right now, the best course of action is to manage your own HTTP calls to Wistia rather than rely on our client library.

**Comment by kylefox** [comment]
@lpommers Fair enough. You might consider removing it from [your documentation](https://wistia.com/support/developers/data-api).

**Comment by kylefox** [comment]
As well as adding a note to the README that this is unmaintained.
